src/main/java/software/amazon/cloudformation/resource/ResourceTypeSchema.java

```diff
@@ -153,6 +149,57 @@ public ResourceTypeSchema(Schema schema) {
             });
             return null;
         });
+
+        this.unprocessedProperties.computeIfPresent("tagging", (k, v) -> {
+            ((Map<?, ?>) v).forEach((key, value) -> {
+                if (!key.equals("tagProperty") && !key.equals("tagPropertyType")) {
```

Comment:
I would suggest you only do that for defined the property name.  It could cause issue or have unexcepted result if there are unknown properties.

Reply:
that make sense, changed it